src/Directory.Build.props

```diff
@@ -42,10 +29,20 @@
     <DefaultTargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard2.1;net6.0</DefaultTargetFrameworks>
     <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
     <AutomaticallyUseReferenceAssemblyPackages>false</AutomaticallyUseReferenceAssemblyPackages>
-    <!-- Pack pdb symbol files -->
-    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
   </PropertyGroup>
 
+  <PropertyGroup Label="SourceLink">
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <!-- NB: Don't generate symbol packages. Symbols and sources are packed into the dlls. -->
+    <IncludeSymbols>false</IncludeSymbols>
+    <DebugType>embedded</DebugType>
+    <EmbedAllSources>true</EmbedAllSources>
```

Comment:
Ok, just checking I understand it correctly: without this line we'd get the file names in exception stack traces, but Go To Definition in IDEs wouldn't work?

Reply:
See https://github.com/riganti/dotvvm/pull/1346#issuecomment-1099103661 for some explanation.

I toyed for a bit with the error page: Without `DebugType=embedded`, we do get filenames but we **don't have line positions** because we **don't have access to the pdb** (nobody does, it isn't copied to `bin/`).

Our error pages have been broken for a while (apparently since .NET 3.0), we just didn't notice, I guess.